### PR TITLE
fix:1784 - Post search input bug

### PR DESCRIPTION
**Pull request by PedroFonsecaDEV**
## Issue This PR Addresses
Fixes: #1784 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

#### Before:
Screen Shot 2021-02-18 at 11 33 49 PM

#### After:
Screen Shot 2021-02-19 at 12 16 38 AM

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
+  createStyles({
+    input: {
+      fontSize: '1.6rem',
+      '&:hover': {
+        border: '1px solid',
+        borderColor: theme.palette.primary.main,
+      },
+      '&:focus': {
+        border: '2px solid',
+        borderColor: theme.palette.primary.main,
```
Comment (DukeManh) [inline src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx:19]: I notice when focused, the input box size increased by 2 pixels. Could you put a `box-sizing: border-box' here to make it consistent?

File: src/frontend/next/src/components/SearchInput/PostSearchInput.tsx [hunk 2]
```diff
@@ -1,48 +1,50 @@
...
+  createStyles({
+    input: {
+      fontSize: '1.6rem',
+      '&:hover': {
+        border: '1px solid',
+        borderColor: theme.palette.primary.main,
+      },
+      '&:focus': {
+        border: '2px solid',
+        borderColor: theme.palette.primary.main,
```
Comment (DukeManh) [inline src/frontend/next/src/components/SearchInput/PostSearchInput.tsx:19]: Same here?

**Review by DukeManh — CHANGES_REQUESTED** [review]

File: src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx (hunk 1 above)
Comment (PedroFonsecaDEV) [reply src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx:19]: Let's keep this for now. The release is tomorrow and then we will be moving to a new UI. So let's just make it functional for now.

File: src/frontend/next/src/components/SearchInput/PostSearchInput.tsx (hunk 2 above)
Comment (PedroFonsecaDEV) [reply src/frontend/next/src/components/SearchInput/PostSearchInput.tsx:19]: Let's keep this for now. The release is tomorrow and then we will be moving to a new UI. So let's just make it functional for now.

**Review by foxviet — APPROVED** [review]

**Review by izhuravlev — APPROVED** [review]
LGTM @PedroFonsecaDEV - not sure if rebase is needed

**Review by birtony — APPROVED** [review]

File: src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx (hunk 1 above)
Comment (DukeManh) [reply src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx:19]: Cool, we can always change it later.

**Review by DukeManh — APPROVED** [review]

**Review by chrispinkney — APPROVED** [review]

**Pull request merged**